data/json/item_actions.json

```
@@ -969,6 +969,11 @@
     "id": "WEED_CAKE",
     "name": { "str": "Consume" }
   },
+  {
+	"type": "item_action",
+	"id": "WEED_DRINK",
+	"name": { "str": "Drink" }
```

```suggestion
	"name": { "ctxt": "verb", "str": "Drink" }
```
Some languages have difference words for the noun and verb of "drink", this ensures they can be separately translated.